generics/services/kendra-service.js

```diff
@@ -7,56 +7,112 @@
 
 //dependencies
 
-let urlPrefix = 
-process.env.KENDRA_SERIVCE_HOST + 
-process.env.KENDRA_SERIVCE_BASE_URL +
-process.env.URL_PREFIX; 
+let apiBaseUrl =
+    process.env.KENDRA_SERIVCE_HOST +
+    process.env.KENDRA_SERIVCE_BASE_URL +
+    process.env.URL_PREFIX;
 
 const request = require('request');
+const fs = require('fs');
 
 /**
-  * Get platform user roles
-  * @function
-  * @name platformUserProfile
-  * @returns {Promise} returns a promise.
-*/
-
-var getPreSignedUrl = function ( token,body,endpoint ) {
-
-    const getPresignedUrl = 
-    urlPrefix + endpoint;
-    
+ * To upload file to cloud 
+ * @name uploadFileToCloud
+ * @param {String} filePath filePath of the file to upload
+ * @param {String} uploadPath - location of bucket where to upload
+ * @param {String} bucketName - name of the bucket
+ * @param {String} token - logged in user token
+ * @param {String} endpoint - endpoint of the api 
+ * @returns {json} - upload file details
+ */
+function uploadFileToCloud(filePath, uploadPath, bucketName, token, endpoint) {
```

Comment:
@rakeshSgr - Parameters which are common for whole service like bucketName and selected cloud service should not be passed while calling functions and should be accessed from env only

Reply:
resolved